Makefile

```diff
@@ -19,35 +17,25 @@ tmp:
 
 # Run the test suite, optionally with coverage
 test: tmp
-	pytest tests/unit -c tests/unit/pytest.ini
+	hatch run test:unit
 smoke: tmp
-	pytest tests/unit/test_smoke.py -c tests/unit/pytest.ini
+	hatch run test:smoke
 coverage: tmp
-	coverage run --source=fmf,bin -m py.test -c tests/unit/pytest.ini tests
-	coverage report
-	coverage annotate
+	hatch run cov:cov
 
 
 # Build documentation, prepare man page
 docs: man
-	cd docs && make html
-man: source
+	hatch run docs:html
+man:
```

Comment:
Is it just me or are you getting similar error?

```
$ make man
cp docs/header.txt /home/user/Repos/fmf/tmp/man.rst
tail -n+7 README.rst >> /home/user/Repos/fmf/tmp/man.rst
rst2man /home/user/Repos/fmf/tmp/man.rst > /home/user/Repos/fmf/tmp/fmf-/fmf.1
/bin/sh: line 1: /home/user/Repos/fmf/tmp/fmf-/fmf.1: No such file or directory
make: *** [Makefile:33: man] Error 1
```

Reply:
Also noticed, `tarball: man` doesn't do much because the man files are created in a temp dir that is not included in the `sdist`. We should revisit this later